### PR TITLE
Feature/63

### PR DESCRIPTION
**Pull request by uedamiku**
注文時にログイン中ユーザの情報が自動入力されるようにしました。

### CODE REVIEW

File: src/main/java/com/example/controller/ConfirmOrderController.java [hunk 1]
```diff
@@ -1,12 +1,15 @@
 package com.example.controller;
 
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.core.annotation.AuthenticationPrincipal;
 import org.springframework.stereotype.Controller;
 import org.springframework.ui.Model;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.RequestMapping;
 
+import com.example.domain.LoginUser;
```
Comment (tkeshun) [inline src/main/java/com/example/controller/ConfirmOrderController.java:10]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Extra separation in import group before 'com.example.domain.LoginUser'

Comment (tkeshun) [inline src/main/java/com/example/controller/ConfirmOrderController.java:10]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Wrong lexicographical order for 'com.example.domain.LoginUser' import. Should be before 'org.springframework.web.bind.annotation.RequestMapping'.

File: src/main/java/com/example/controller/ConfirmOrderController.java [hunk 2]
```diff
@@ -1,12 +1,15 @@
...
 import com.example.domain.Order;
+import com.example.domain.User;
```
Comment (tkeshun) [inline src/main/java/com/example/controller/ConfirmOrderController.java:12]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Wrong lexicographical order for 'com.example.domain.User' import. Should be before 'org.springframework.web.bind.annotation.RequestMapping'.

File: src/main/java/com/example/controller/ConfirmOrderController.java [hunk 3]
```diff
@@ -24,14 +27,26 @@ public class ConfirmOrderController {
 
 	/**
 	 * 注目確認画面を表示します.
-	 * 	
+	 *
```
Comment (tkeshun) [inline src/main/java/com/example/controller/ConfirmOrderController.java:30]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

File: src/main/java/com/example/controller/ConfirmOrderController.java [hunk 4]
```diff
@@ -24,14 +27,26 @@ public class ConfirmOrderController {
...
 	 * 注目確認画面を表示します.
-	 * 	
+	 * 
 	 * @param orderId 注文の主キー
 	 * @return 注文確認画面
 	 */
 	@GetMapping("/to-confirm-order")
-	public String toConfirmOrder(Integer orderId, Model model, SendOrderForm sendOrderForm) {
-		Order order = confirmOrderService.showOrder(orderId);
+	public String toConfirmOrder(Integer orderId, Model model, SendOrderForm sendOrderForm,
```
Comment (tkeshun) [inline src/main/java/com/example/controller/ConfirmOrderController.java:35]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

File: src/main/java/com/example/controller/ConfirmOrderController.java [hunk 5]
```diff
@@ -24,14 +27,26 @@ public class ConfirmOrderController {
...
+			@AuthenticationPrincipal LoginUser loginUser) {
```
Comment (tkeshun) [inline src/main/java/com/example/controller/ConfirmOrderController.java:36]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

File: src/main/java/com/example/controller/ConfirmOrderController.java [hunk 6]
```diff
@@ -24,14 +27,26 @@ public class ConfirmOrderController {
...
+		User user = loginUser.getUser();
```
Comment (tkeshun) [inline src/main/java/com/example/controller/ConfirmOrderController.java:37]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/controller/ConfirmOrderController.java:37]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 16, expected level should be 4.

File: src/main/java/com/example/controller/ConfirmOrderController.java [hunk 7]
```diff
@@ -24,14 +27,26 @@ public class ConfirmOrderController {
...
+		model.addAttribute("user", user);
```
Comment (tkeshun) [inline src/main/java/com/example/controller/ConfirmOrderController.java:38]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/controller/ConfirmOrderController.java:38]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 16, expected level should be 4.

File: src/main/java/com/example/controller/ConfirmOrderController.java [hunk 8]
```diff
@@ -24,14 +27,26 @@ public class ConfirmOrderController {
...
+		sendOrderForm.setId(user.getId());
```
Comment (tkeshun) [inline src/main/java/com/example/controller/ConfirmOrderController.java:39]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/controller/ConfirmOrderController.java:39]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 16, expected level should be 4.

File: src/main/java/com/example/controller/ConfirmOrderController.java [hunk 9]
```diff
@@ -24,14 +27,26 @@ public class ConfirmOrderController {
...
+		sendOrderForm.setDestinationName(user.getName());
```
Comment (tkeshun) [inline src/main/java/com/example/controller/ConfirmOrderController.java:40]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/controller/ConfirmOrderController.java:40]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 16, expected level should be 4.

File: src/main/java/com/example/controller/ConfirmOrderController.java [hunk 10]
```diff
@@ -24,14 +27,26 @@ public class ConfirmOrderController {
...
+		sendOrderForm.setDestinationEmail(user.getEmail());
```
Comment (tkeshun) [inline src/main/java/com/example/controller/ConfirmOrderController.java:41]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/controller/ConfirmOrderController.java:41]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 16, expected level should be 4.

File: src/main/java/com/example/controller/ConfirmOrderController.java [hunk 11]
```diff
@@ -24,14 +27,26 @@ public class ConfirmOrderController {
...
+
+		sendOrderForm.setDestinationZipcode(user.getZipcode());
```
Comment (tkeshun) [inline src/main/java/com/example/controller/ConfirmOrderController.java:43]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/controller/ConfirmOrderController.java:43]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 16, expected level should be 4.

File: src/main/java/com/example/controller/ConfirmOrderController.java [hunk 12]
```diff
@@ -24,14 +27,26 @@ public class ConfirmOrderController {
...
+		sendOrderForm.setDestinationPref(user.getPref());
```
Comment (tkeshun) [inline src/main/java/com/example/controller/ConfirmOrderController.java:44]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/controller/ConfirmOrderController.java:44]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 16, expected level should be 4.

File: src/main/java/com/example/controller/ConfirmOrderController.java [hunk 13]
```diff
@@ -24,14 +27,26 @@ public class ConfirmOrderController {
...
+		sendOrderForm.setDestinationMunicipalities(user.getMunicipalities());
```
Comment (tkeshun) [inline src/main/java/com/example/controller/ConfirmOrderController.java:45]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/controller/ConfirmOrderController.java:45]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 16, expected level should be 4.

File: src/main/java/com/example/controller/ConfirmOrderController.java [hunk 14]
```diff
@@ -24,14 +27,26 @@ public class ConfirmOrderController {
...
+		sendOrderForm.setDestinationAddress(user.getAddress());
```
Comment (tkeshun) [inline src/main/java/com/example/controller/ConfirmOrderController.java:46]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/controller/ConfirmOrderController.java:46]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 16, expected level should be 4.

File: src/main/java/com/example/controller/ConfirmOrderController.java [hunk 15]
```diff
@@ -24,14 +27,26 @@ public class ConfirmOrderController {
...
+		sendOrderForm.setDestinationTel(user.getTelephone());
```
Comment (tkeshun) [inline src/main/java/com/example/controller/ConfirmOrderController.java:47]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/controller/ConfirmOrderController.java:47]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 16, expected level should be 4.

File: src/main/java/com/example/controller/ConfirmOrderController.java [hunk 16]
```diff
@@ -24,14 +27,26 @@ public class ConfirmOrderController {
...
 
+		Order order = confirmOrderService.showOrder(orderId);
```
Comment (tkeshun) [inline src/main/java/com/example/controller/ConfirmOrderController.java:49]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/controller/ConfirmOrderController.java:49]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 16, expected level should be 4.

File: src/main/java/com/example/controller/SendOrderController.java [hunk 17]
```diff
@@ -1,13 +1,15 @@
...
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.core.annotation.AuthenticationPrincipal;
 import org.springframework.stereotype.Controller;
 import org.springframework.ui.Model;
 import org.springframework.validation.BindingResult;
 import org.springframework.validation.annotation.Validated;
 import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RequestMapping;
 
+import com.example.domain.LoginUser;
```
Comment (tkeshun) [inline src/main/java/com/example/controller/SendOrderController.java:12]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Extra separation in import group before 'com.example.domain.LoginUser'

Comment (tkeshun) [inline src/main/java/com/example/controller/SendOrderController.java:12]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Wrong lexicographical order for 'com.example.domain.LoginUser' import. Should be before 'org.springframework.web.bind.annotation.RequestMapping'.

**Review by tkeshun — COMMENTED** [review]
Remaining comments which cannot be posted as a review comment to avoid GitHub Rate Limit

<details>
<summary>checkstyle</summary>

[src/main/java/com/example/controller/SendOrderController.java|40 col 1|](http://github.com/ecsite-project/ec-202304c/blob/0803988a5c5ebd7f64beda26b6d5266c1d6d866d/src/main/java/com/example/controller/SendOrderController.java#L40) Line contains a tab character.
[src/main/java/com/example/controller/SendOrderController.java|41 col 1|](http://github.com/ecsite-project/ec-202304c/blob/0803988a5c5ebd7f64beda26b6d5266c1d6d866d/src/main/java/com/example/controller/SendOrderController.java#L41) Line contains a tab character.
[src/main/java/com/example/controller/SendOrderController.java|44 col 1|](http://github.com/ecsite-project/ec-202304c/blob/0803988a5c5ebd7f64beda26b6d5266c1d6d866d/src/main/java/com/example/controller/SendOrderController.java#L44) Line contains a tab character.
[src/main/java/com/example/controller/SendOrderController.java|44 col 25|](http://github.com/ecsite-project/ec-202304c/blob/0803988a5c5ebd7f64beda26b6d5266c1d6d866d/src/main/java/com/example/controller/SendOrderController.java#L44) 'if' child has incorrect indentation level 24, expected level should be 6.
[src/main/java/com/example/controller/SendOrderController.java|51|](http://github.com/ecsite-project/ec-202304c/blob/0803988a5c5ebd7f64beda26b6d5266c1d6d866d/src/main/java/com/example/controller/SendOrderController.java#L51) Line is longer than 100 characters (found 107).
[src/main/java/com/example/controller/SendOrderController.java|51 col 1|](http://github.com/ecsite-project/ec-202304c/blob/0803988a5c5ebd7f64beda26b6d5266c1d6d866d/src/main/java/com/example/controller/SendOrderController.java#L51) Line contains a tab character.
[src/main/java/com/example/controller/SendOrderController.java|51 col 25|](http://github.com/ecsite-project/ec-202304c/blob/0803988a5c5ebd7f64beda26b6d5266c1d6d866d/src/main/java/com/example/controller/SendOrderController.java#L51) 'if' child has incorrect indentation level 24, expected level should be 6.
</details>

**Review by yousuke-murayama — APPROVED** [review]
問題ないと思います。

File: src/main/java/com/example/controller/SendOrderController.java [hunk 18]
```diff
@@ -35,17 +37,18 @@ public class SendOrderController {
 	 * @return 注文完了画面
 	 */
 	@PostMapping("/send-order")
-	public String sendOrder(@Validated SendOrderForm form, BindingResult rs, Model model) {
+	public String sendOrder(@Validated SendOrderForm form, BindingResult rs, Model model,
```
Comment (tkeshun) [inline src/main/java/com/example/controller/SendOrderController.java:40]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

File: src/main/java/com/example/controller/SendOrderController.java [hunk 19]
```diff
@@ -35,17 +37,18 @@ public class SendOrderController {
...
+			@AuthenticationPrincipal LoginUser loginUser) {
```
Comment (tkeshun) [inline src/main/java/com/example/controller/SendOrderController.java:41]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

File: src/main/java/com/example/controller/SendOrderController.java [hunk 20]
```diff
@@ -35,17 +37,18 @@ public class SendOrderController {
...
 
 		if (("").equals(form.getDestinationEmail())) {
-			rs.rejectValue("destinationEmail", null, "メールアドレスは必須です");
+			rs.rejectValue("destinationEmail", null, "メールアドレスを入力してください");
```
Comment (tkeshun) [inline src/main/java/com/example/controller/SendOrderController.java:44]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/controller/SendOrderController.java:44]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'if' child has incorrect indentation level 24, expected level should be 6.

File: src/main/java/com/example/controller/SendOrderController.java [hunk 21]
```diff
@@ -35,17 +37,18 @@ public class SendOrderController {
...
 		}
 		
 		if (("").equals(form.getDestinationPref()) || ("").equals(form.getDestinationMunicipalities()) || ("").equals(form.getDestinationAddress())) {
 			rs.rejectValue("destinationPref", null, "住所を入力してください");
 		}
 		if (rs.hasErrors()) {
-			return confirmOrderController.toConfirmOrder(form.getId(), model, form);
+			return confirmOrderController.toConfirmOrder(form.getId(), model, form, loginUser);
```
Comment (tkeshun) [inline src/main/java/com/example/controller/SendOrderController.java:51]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line is longer than 100 characters (found 107).

Comment (tkeshun) [inline src/main/java/com/example/controller/SendOrderController.java:51]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/controller/SendOrderController.java:51]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'if' child has incorrect indentation level 24, expected level should be 6.

**Pull request merged**